images/linux/scripts/installers/aws.sh

```diff
@@ -6,13 +6,24 @@
 
 # Source the helpers
 source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/os.sh
 
-# Install the AWS CLI
-curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
-unzip awscli-bundle.zip
-./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
-rm awscli-bundle.zip
-rm -rf awscli-bundle
+# Install the AWS CLI v1 on Ubuntu16 and Ubuntu18, and AWS CLI v2 on Ubuntu20
+if isUbuntu20 ; then
+    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
There is much sense to have two condition blocks with almost identical steps. Perhaps you can move installation logic to separate function?
```
if isUbuntu20; then
    install_aws "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" "/usr/local/aws-cli" "/usr/local/bin"
fi

if isUbuntu20; then
    install_aws "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" "/usr/local/aws" "/usr/local/bin/aws"
fi
```

Reply:
Unfortunately, we can't use same approach for both AWL CLI V1 and V2 installation. V2 contains some changes in source code, that's why we were forced to use two condition blocks.